docker-extension/docker-compose.yml

```diff
@@ -3,9 +3,9 @@ version: '3'
 services:
   headlamp:
     image: ghcr.io/kinvolk/headlamp:v0.13.0
-    command: ["--kubeconfig","/headlamp/config/config"]
+    command: ["--dev","--kubeconfig","/headlamp/config/config"]
```

Comment:
This is because the docker-desktop extension runs the frontend and the backend service separately. Right now the port is also changed to `64446` (to use a not-so-common port there by reducing the probability of port clash) without enabling the `dev` mode the frontend is getting `CORS` issue.

Reply:
What's the CORS issue? Dev mode disables all the CORS/security features. I'm not sure if that's what we want to do.